app/src/main/kotlin/com/intive/picover/common/di/AppModule.kt

```diff
@@ -0,0 +1,22 @@
+package com.intive.picover.common.di
+
+import com.google.firebase.storage.FirebaseStorage
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+import kotlinx.coroutines.Dispatchers
+
+@Module
+@InstallIn(SingletonComponent::class)
+object AppModule {
+
+	@Provides
+	@Singleton
+	fun provideFirebaseStoreInstance() = FirebaseStorage.getInstance()
+
+	@Provides
+	@Singleton
+	fun provideIoDispatcher() = Dispatchers.IO
```

Comment:
I would create an issue for providing a class for handling dispatchers for tests and normal cases, for now it is ok for me for providing it like this, but will be good to document a future approach :)

Reply:
Please see the issue #110